deploy/src/deployment-servicemesh.yaml

```diff
@@ -64,6 +64,13 @@ spec:
         olm.relatedImage.v2_2.proxyv2: ${OSSM_PROXY_IMAGE}
         olm.relatedImage.v2_2.wasm-cacher: ${OSSM_PILOT_IMAGE}
         olm.relatedImage.v2_2.rls: ${OSSM_RATELIMIT_IMAGE}
+        # 2.3 images
+        olm.relatedImage.v2_3.cni: ${OSSM_CNI_IMAGE}
+        olm.relatedImage.v2_3.grafana: ${OSSM_GRAFANA_IMAGE}
+        olm.relatedImage.v2_3.pilot: ${OSSM_PILOT_IMAGE}
+        olm.relatedImage.v2_3.prometheus: ${OSSM_PROMETHEUS_IMAGE}
+        olm.relatedImage.v2_3.proxyv2: ${OSSM_PROXY_IMAGE}
+        olm.relatedImage.v2_3.rls: ${OSSM_RATELIMIT_IMAGE}
```

Comment:
I don't know where we set these variables, but using the same variable for both 2.2 and 2.3 is surely wrong. You probably need to change the values for 2.2 to be more like those for 2.1 (with the actual image name + `:${OSSM_22}` at the end) and probably set the {{OSSM_22}} variable somewhere.

Reply:
These variables are set in CPaaS, but you're right that these variables cannot be the same as for 2.2.

@jwendell do you have any suggestion? Can I name these variables to `OSSM_CNI_2_3_*`?

Reply:
I think you need to keep the current name, and instead change the 2.2 values. For example:
```
olm.relatedImage.v2_2.rls: ${OSSM_RATELIMIT_IMAGE}
```
must become:
```
olm.relatedImage.v2_2.rls: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-ratelimit-rhel8:${OSSM_22}
```

Reply:
Done